config.yaml

```diff
@@ -6,11 +6,46 @@ options:
     description: The number of webserver worker processes for handling requests.
   webserver_threads:
     type: int
-    description: >
-      The number of worker threads for handling requests. Run each webserver worker with the specified number of threads.
+    description: Run each webserver worker with the specified number of threads.
   webserver_keepalive:
     type: int
-    description: The number of seconds for webserver to wait for requests on a Keep-Alive connection.
+    description: Time in seconds for webserver to wait for requests on a Keep-Alive connection.
   webserver_timeout:
     type: int
-    description: Webserver workers silent for more than this many seconds are killed and restarted.
+    description: Time in seconds to kill and restart silent webserver workers.
+  flask_env:
+    type: string
+    description: What environment the Flask app is running in, by default it's 'production'.
+  flask_debug:
+    type: boolean
+    description: Whether Flask debug mode is enabled.
```

Comment:
s/Whether Flask debug mode is enabled/Flask debug mode enabled

Reply:
I think I will keep the current version for it's a complete sentences.